metricflow/test/query/test_query_parser.py

```diff
@@ -16,11 +18,81 @@
 from metricflow.test.test_utils import as_datetime
 from metricflow.test.time.metric_time_dimension import MTD
 from metricflow.time.time_granularity import TimeGranularity
+from metricflow.test.fixtures.model_fixtures import query_parser_from_yaml
 
 logger = logging.getLogger(__name__)
 
 
-def test_query_parser(query_parser: MetricFlowQueryParser) -> None:  # noqa: D
+BOOKINGS_YAML = textwrap.dedent(
+    """\
+    data_source:
+      name: bookings_source
+
+      sql_query: |
+        -- User Defined SQL Query
+        SELECT * FROM $bookings_source_table
```

Comment:
If you like you can put in a string literal name for the table, there's no underlying data involved here. Doesn't matter, either way it'll likely fail if anything tries to run a query.